_episodes/09-pull-requests.md

```diff
@@ -133,17 +211,18 @@ $ git commit -m "Added file on france"
 {: .bash}
 
 ~~~
-[master 79a312a] Added file on france
+[addFrance 79a312a] Added file on france
  1 file changed, 2 insertions(+), 2 deletions(-)
 ~~~
 {: .output}
 
 Now we can push those changes to our forked version of the repo.
 Since we don't have permissions to the upstream repo or we want to suggest these changed be
```

Comment:
I think it would help to remove the first line. Why we're pushing to the fork should probably be be explained. This assertion that we're going to do so appears abruptly. I like the way you modified the second sentence here. I think the first half of that sentence lays out the two main motivations for using PR's to suggest changes. The reason that we're pushing to the fork is sort of a by product of setting up a PR but it isn't framed that way. Right now there's a gap between identifying a situation in which you want to make a PR and pushing to the leaner's fork. 

I'll propose another iteration:

"In some cases we may not have permission to push changes directly to the upstream repo or we might like our changes to be reviewed regardless of permissions, so we'll create a `pull request`. A `pull request` is a **request** for a member of the upstream repository to **pull** our changes into the upstream repository from a `fork`, allowing them to request further changes/improvement and make comments on the changes before doing so. In order to create a `pull request`, we must push our new branch containing the changes we'd like to submit to the remote linked to our fork, `origin`, on GitHub."